src/Firehose.Web/Authors/JeromeGiacomini.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Firehose.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using System.Web;
+
+namespace Firehose.Web.Authors
+{
+    public class JeromeGiacomini : IAmACommunityMember
+    {
+        public string FirstName => "Jérôme";
+
+        public string LastName => "Giacomini";
+
+        public string StateOrRegion => "Paris, France";
+
+        public string EmailAddress => "jerome.giacomini@gmail.com";
+
+        public string ShortBioOrTagLine => "is a Xamarin Enthusiast, co-author of a book on Xamarin";
+
+        public Uri WebSite => new Uri("https://jeromegiacomini.net/Blog/");
+
+        public string TwitterHandle => "jeromegiacomini";
+
+        public string GitHubHandle => "jgiacomini";
+
+        public string GravatarHash => "";
+
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("https://jeromegiacomini.net/Blog/feed/"); }
+        }
+
+        public GeoPosition Position => new GeoPosition(48.8704842, 2.3449646);
+
+        public string FeedLanguageCode => "fr";
+
+        public bool Filter(SyndicationItem item) =>
```

Comment:
Remove the filter. We already filter for Xamarin by default and you don't implement `IFilterMyBlogPosts`.

Reply:
If you worried about my activity maybe I can reassure you.

The last year I had 2 blogs : 
- https://jeromegiacomini.net/Blog/ (my perso)
- https://blogs.infinitesquare.com/users/jgiacomini (my pro blog)
I wrote also a book on Xamarin : https://www.editions-eni.fr/supports-de-cours/livre/xamarin-developpez-vos-applications-multiplateformes-pour-ios-android-et-windows-9782409007477

This year each post I will wrote on my pro blog=> i will also put them on perso blog. :)

Reply:
My book is now finished I will have more time to write article. :)